db/schema.sql

```diff
@@ -13,7 +13,8 @@ CREATE TABLE IF NOT EXISTS enrollments (
 );
 
 CREATE TABLE IF NOT EXISTS students (
-  id integer REFERENCES enrollments,
+  id SERIAL PRIMARY KEY,
```

Comment:
@magee Added `id` field. Does this seem reasonable?

Reply:
Perfect!